### ISSUE TITLE
Add version picker in demo

### ISSUE THREAD

**Issue by JackFly26** [kognise/water.css#27]
I think it would be a good idea for the demo to have a link to the cdn of the current theme that the page is on. Like when it's on light, the link to the light cdn, and the same for dark.

**Comment by kognise** [comment]
Or possibly a code block with the code to include the current theme in your website? I'm working on making it less of a demo and more of a landing page

**Comment by kylejrp** [comment]
We would love for contributions on this issue! Here's what would need to be done:

- In `index.html`, add a `<code>` element containing the latest link to the CDN (`<link rel='stylesheet' href='https://cdn.jsdelivr.net/gh/kognise/water.css@latest/dist/dark.min.css'>`)
- In `script.js`, modify the existing function to also change the `<code>` block so that it changes to `light.min.css` or `dark.min.css` when appropriate.

**Comment by kylejrp** [comment]
This would also be a good opportunity to the code from https://github.com/kognise/water.css/issues/25#issuecomment-482664700:

```
<link id="stylesheet" rel="stylesheet" href="dist/light.css">
<link id="stylesheet" rel="stylesheet" href="dist/dark.css" media="(prefers-color-scheme: dark)">
```

**Comment by jonaskuske** [comment]
Also posting this here as it's relevant:

Made an interface for selecting and copying the code required for `water.css`.
It currently includes options discussed in other issues (npm links instead of GitHub hosted, auto-theme based on `prefers-color-scheme`, modern build with CSS variables) but those can be easily removed of course. :)
https://codepen.io/21stCenturyJonas/full/JqjPgL

Let me know how you feel about this!

**Comment by kylejrp** [comment]
@jonaskuske, this seriously awesome work! I love what you've done so far. This really solves some of the problems we've been discussing about how to make water.css easy to use.

I think we would want to make it easy to copy and paste without reading into the advanced options. The ideal interface order in my head would be:

**Get `water.css` now!**
Paste this into the <head> of your HTML:      [Copy to clipboard]
```<link rel="stylesheet" href="https://unpkg.com/water.css/dark.min.css">```
x Dark     o Light
<details>
<summary>Advanced Options</summary>

|Option|Description|
|-|-|
|[x] Allow user setting of `prefers-color-scheme`?|💡 The default theme will be dark, but if the user's device is set to light mode, this setting will be respected.|
|[ ] Support Internet Explorer and other legacy browsers?|💡 The default version of water.css uses CSS variables to switch between dark and light mode. If not all of your targeted browsers support these, use the legacy version. You can find details about browser support here.|

</details>

Then, most users can just copy whatever our preferred sensible defaults are without having to think about it. Most people just want to copy and paste the defaults anyways.

Maybe there could even be a listing of the total sizes of the bundle (before/after gzip?)

But even as is, your interface absolutely rocks. I think you should make a pull request with the current state of water.css without the advanced options for now so we can get this on the demo page! After we figure out #70 and #25, we can add those advanced options in.

**Comment by kylejrp** [comment]
Maybe it would be good to show a browser compatibility matrix for the options too? I dunno, there's a lot of possibilities in this UI. You've got my mind racing!

**Comment by jonaskuske** [comment]
Agree on your reorder proposal, having the installation code readily available right at the beginning definitely makes sense!

CSS & Sass compilation is only in Beta there, but we might be able to simply add [bundlephobia badges](https://badgen.net/#bundlephobia) once we're up on npm.

As for browser support: The proposed new default builds would only support modern browsers, everything else would keep the previous, broader browser support as it's just "classic" CSS. We could better highlight which browsers do support CSS vars (or maybe rather which don't?) but the support matrix shouldn't be that hard to communicate I think. :)

**Comment by kylejrp** [comment]
I love the bundlephobia badges!

I'm not too worried about legacy browser support by default as long as it's communicated clearly and is still an option. Maybe @kognise can chime in on opinions on default browser support and how we communicate that.

**Comment by kylejrp** [comment]
Now that #71 is merged we can move forward with adding the version picker to the demo.

**Comment by kognise** [comment]
This has been fixed in 2.0 🎉
